### ISSUE TITLE
Pages Rendering With Wrong Webfont

### ISSUE THREAD

**Issue by lahdekorpi** [fzaninotto/screenshot-as-a-service#34]
I'm having some weird issues with wrong (from previous sites) webfonts are shown.

Test case:
1. Boot up the service
2. Load http://x.local that uses a .woff: "Montserrat-Regular"
3. Screenshot works as it should be with the correct font
4. Load http://y.local that uses a .woff: "Sansation-Regular"
5. Screenshot shows the site using "Montserrat-Regular" instead of the correct one
6. `killall phantomjs`
7. Load http://y.local again
8. Screenshot works as it should be with the correct font
9. Load http://x.local again
10. Screenshot shows the site using "Sansation-Regular" instead of the correct one

**TL;DR: Once a webfont is loaded, all pages show up with that one.**

**Comment by slorber** [comment]
Seems to have the same problem here: we noticed phantomJS took 2 different screenshots of the same website (and it looks like the difference is font)
